lib/holidays/date_calculator/lunisolar.rb

```diff
@@ -0,0 +1,218 @@
+module Holidays
+  module DateCalculator
+    # Copied from Korean Lunar/Solar Calendar Library, by sunsidew
+    # https://github.com/sunsidew/ruby_lunardate
```

Comment:
Yes, of course. I'll make sure I handle this when I perform the updates you listed below. Thank you!